server/sonar-server/src/main/java/org/sonar/server/qualityprofile/QProfileCopier.java

```diff
@@ -42,46 +44,44 @@
   private final QProfileFactory factory;
   private final QProfileBackuper backuper;
   private final TempFolder temp;
+  private final QProfileWsSupport qProfileWsSupport;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove this unused "qProfileWsSupport" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1068)